packages/blade/src/components/Icons/iconMap.ts

```diff
@@ -8,10 +8,10 @@ import DollarIconComponent from './DollarIcon';
 import EyeIconComponent from './EyeIcon';
 import EyeOffIconComponent from './EyeOffIcon';
 import RupeeIconComponent from './RupeeIcon';
-import type { IconProps } from '.';
+import type { IconComponentType } from '.';
```

Comment:
```suggestion
import type { IconComponent } from '.';
```

nit: Let's keep it as `IconComponent` as while defining and importing we already use `type` keyword